src/states/node.rs

```diff
@@ -1593,6 +1616,72 @@ impl Node {
         result.with_value(Ok(()))
     }
 
+    fn handle_rt_req(&mut self,
+                     msg_id: MessageId,
+                     digest: sha256::Digest,
+                     src: Authority<XorName>,
+                     dst: Authority<XorName>)
+                     -> Evented<Result<(), RoutingError>> {
+        // TODO: Compare the hash and don't send anything if he requester is up-to-date.
+        let mut result = Evented::empty();
+        let sections = self.peer_mgr.pub_ids_by_section(self.full_id.public_id());
```

Comment:
The duplicated part is basically just:
```rust
sha256::hash(serialisation::serialise(&sections)?)
```
Not sure if that justifies a separate method. And I was thinking of another deduplication, using `pub_ids_by_section` in other places in the peer manager, so it wouldn't make sense to compute the hash in there either.